apps/operator/src/components/pages/protected/documents/editor.tsx

```diff
@@ -41,9 +40,7 @@ export const TemplateEditor = ({ id }: { id: string }) => {
   // setup the query to get the template data
   const variables = { getTemplateId: id || '' }
   const [templateData] = useGetTemplateQuery({ variables })
-  const uischema =
-    templateData?.data?.template.uischema ||
-    Generate.uiSchema(templateData.data?.template.jsonconfig)
+  const jsonSchema = templateData.data?.template.jsonconfig
```

Comment:
```suggestion
  const jsonSchema = templateData.data?.template.uischema
```

The `uischema` is different from the jsonschema, `JsonForms` allowed this to be autogenerated, I don't know if react-forms exposes this functionality but if not, it should just always be pulled from the db and used if not empty to render the form.